spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -875,6 +876,13 @@ void testTwoThreadsRemoveAndObtainSameLockSimultaneously(RedisLockType testRedis
 		registry.destroy();
 	}
 
+	@Test
+	void testInitialiseWithCustomExecutor() {
+		RedisLockRegistry redisLockRegistry = new RedisLockRegistry(redisConnectionFactory, "registryKey");
+		redisLockRegistry.setRedisLockType(RedisLockType.PUB_SUB_LOCK);
+		redisLockRegistry.setExecutor(Executors.newFixedThreadPool(2));
```

Comment:
Wrap, please, `assertThatNoException().isThrownBy(() -> )` to make the reason from this test code.

Reply:
Updated test as reommended. Thanks for the feedback.